search_that_hash/api.py

```diff
@@ -1,26 +1,25 @@
-from search_that_hash.cracker import cracking
 from search_that_hash import config_object
 from search_that_hash import __main__
-
+from search_that_hash.cracker import handler
 import json
 
 
 def return_as_json(hashes):
 
     config = config_object.api_config(hashes)
     config["greppable"] = True
+    cracking_handler = handler.handler(config)
+
+    json = cracking_handler.start()
```

Comment:
json is a keyword in python, as in you cant use the same word as a keyword (like hash is a keyword). It's like doing `print = "hello"` later on you cant do `print("hello")` since you assigned it to a variable not a function pls choose a different name